### ISSUE TITLE
Sonification speech broken?

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/highcharts#25306]
#### Expected behaviour
Expect this demo to read out "Summer" when getting to the plotband.

#### Actual behaviour
The "Summer" speech is missing, garbled, or too fast.

This might be an issue with the parameters sent into the WebSpeechSynthesis API. For example, the wrong voice profile may be chosen (there is some logic there), or the speed/pitch parameters may have been set wrong somehow.

See https://github.com/highcharts/highcharts/blob/master/ts/Extensions/Sonification/SonificationSpeaker.ts for relevant code.

#### Live demo with steps to reproduce
https://www.highcharts.com/demo/highcharts/plotline-context

#### Product version
Latest Core.

#### Affected browser(s)
 Tested in Chrome & Safari

**Comment by oysteinmoseng** [comment]
Note: Setting `language` and `preferredVoice` explicitly works. Platform dependent, though. Might be an issue with identifying a usable voice for the default language. https://jsfiddle.net/2uwy59ct/ 

Some voices are "whisper" effect voices, and those tend to sound pretty garbled when used with high speech rate, so most likely such a voice is selected by accident.
